aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -103,10 +95,6 @@ impl SessionId {
         }
     }
 
-    pub fn prologue(txn: &SignedTransaction) -> Self {
-        Self::prologue_meta(&TransactionMetadata::new(&txn.clone()))
-    }
-
     pub fn prologue_meta(txn_data: &TransactionMetadata) -> Self {
```

Comment:
maybe txn_data -> txn_metadata here and below / in this file (data isn't supposed to be metadata), would also be more consistent with naming in aptos_vm